### ISSUE TITLE
WPF Bug 457740

### ISSUE THREAD

**Issue by rbmoman** [Microsoft/dotnet-framework-early-access#20]
ISSUE: In a master/detail scenario using ADO data, changing the primary key of a master item cleared the display of the corresponding details, with no way to display them thereafter. This is fixed by default for apps targeting .NET Framework 4.7.2, and for all apps that set the app-context switch Switch.System.Windows.Data.DoNotUseFollowParentWhenBindingToADODataRelation to false. [457740, PresentationFramework.dll, Bug]

When I build and run directly under VS2015, the bug remainds, the corresponding details are still cleared.

On the other hand, if I run the application targeting .NET Framework 4.7.2 directly from the Windows operating system, the bug does not happen and the display of the corresponding details is updated correctly.

Would it be possible to fix the problem when running throught Visual Studio?

Thanks for you reply

Raymond

**Comment by preetikr** [comment]
Did you install [.NET Framework 4.7.2 Developer Pack (Early Access)](https://blogs.msdn.microsoft.com/dotnet/2018/03/08/net-framework-4-7-2-developer-pack-early-access-build-3056-is-available/) for VS 2015 scenario to work? This will enable you to select targeting for .NET Framework 4.7.2 in VS2015 before building.

**Comment by rbmoman** [comment]
Yes, I did it before posting here. I followed Preeti Krishna's reply to my issue on this page [https://blogs.msdn.microsoft.com/dotnet/2018/02/05/announcing-net-framework-4-7-2-early-access-build-3052/](url)

Issue
rbmoman
How to: Compile and execute a project with VS 2015 targetting .Net Framework 4.7.2?

Reply
Preeti Krishna - MSFT
You can do so with the Early Access build of .NET Framework 4.7.2 Developer Pack. Details are @ https://blogs.msdn.microsoft.com/dotnet/2018/03/08/net-framework-4-7-2-developer-pack-early-access-build-3056-is-available/#respond

Have you tried to reproduce the bug on your side?

Thanks for your help
Raymond

**Comment by rbmoman** [comment]
Is there a solution for this issue?
Thank you for yours replies
